tests/disas/winch/x64/f32x4_abs/const_avx.wat

```diff
@@ -14,28 +14,29 @@
 ;;       movq    0x10(%r11), %r11
 ;;       addq    $0x10, %r11
 ;;       cmpq    %rsp, %r11
-;;       ja      0x4f
+;;       ja      0x50
```

Comment:
This looks peculiar! This indicates that we might be placing an unnecessary byte which isn't required? Or that the old encoding for these instructions was 1-byte smaller than the encoding from the assembler?

(or maybe a rebase conflict as one of the PRs I landed recently removed some bytes and this might be adding them back in without having rebased on that? unsure)

Reply:
Ok, looking at this again, I think we should just accept this (like you were pointing out when we talked). This is coming because this PR switched the order of the operands as it lowered the instruction and I think this operand swap makes more sense. Take a look at the constructor I tagged below: we pass `src1` and `src2` in the order one would expect. I'm just going to leave this.